### PR TITLE
Addubg Blockbench documentation markdown

### PR DESCRIPTION
**Pull request by williamharmon**
Making a markdown entry for blockbench documentation.

A link to look show how the markdown is displayed on github:
https://github.com/Mojang/minecraft-legends-docs/blob/williamharmon/blockbench-doc/BlockBench.md

### CODE REVIEW

File: BlockBench.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+To use this plugin, you need [Blockbench](https://www.blockbench.net/downloads)
+
+# Installing the plugin
+
+1.  Download the [Minecraft: Legends plugin](https://github.com/Mojang/legends-blockbench-plugin/releases)
+2.	Open Blockbench and using the menu bar select **File** > **Plugins**
+
+![](images/blockbench_doc/image01.png)
+
+3.	Select the load plugin icon (the icon between “Plugins” and the cloud icon). This will open a file dialog. From the file dialog, navigate to where you saved the downloaded plugin, select the plugin, and then select Open.
```
Comment (jsyarrow) [inline BlockBench.md:37]: These steps kind of imply that users should try to open the downloaded zip file from the releases page (which of course is not right).  I think we should be very explicit about unzipping the downloaded release and pointing at the appropriate js file.

Comment (williamharmon) [reply BlockBench.md:37]: Good call!

**Review by jsyarrow — APPROVED** [review]

File: BlockBench.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+* letters 
+
+* numbers
+
+* . (period)
+
+* - (dash)
+
+* _ (underline)
```
Comment (jsyarrow) [inline BlockBench.md:111]:
```suggestion
* letters 
* numbers
* ```.``` (period)
* ```-``` (dash)
* ```_``` (underline) 
```

File: BlockBench.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Be sure to only use the Bedrock Entity Project type when creating projects as the exporter only supports cube geometry.
+```
+
+In this Article you will learn the following:
+
+* How to install Blockbench
+
+* How to install the plugin
+
+* How to export models, animations and textures
```
Comment (jsyarrow) [inline BlockBench.md:19]:
```suggestion
* How to install Blockbench
* How to install the plugin
* How to export models, animations and textures
```

File: BlockBench.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+When you have set your Export Options it will open a folder selection menu where you can select where the files will be exported. You have now successfully exported the following files:
+
+* model
+
+* texture
+
+* material
+
+* animations (if export animations were selected)
+
```
Comment (jsyarrow) [inline BlockBench.md:81]:
```suggestion
* model
* texture
* material
* animations (if export animations were selected)
```

File: BlockBench.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+**Naming conventions**:
+Make sure to only use 
+
+* letters 
+* numbers
+* ```.``` (period)
+* ```-``` (dash)
+* ```_``` (underline) 
+
```
Comment (jsyarrow) [inline BlockBench.md:108]:
```suggestion
* letters 
* numbers
* ```.``` (period)
* ```-``` (dash)
* ```_``` (underline) 
```

File: BlockBench.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+* **Alpha Test**: This check box determines if the material will interpret the alpha channel of the texture as transparency. If not checked, all texels will be opaque.
+* **Export Animations**: If this is checked when you press confirm it will open a secondary dialog where you can select which animations to export.
+
+![](images/blockbench_doc/image07.png)
+
+When you have set your Export Options it will open a folder selection menu where you can select where the files will be exported. You have now successfully exported the following files:
+* model
+* texture
+* material
+* animations (if export animations were selected)
```
Comment (jsyarrow) [inline BlockBench.md:74]:
```suggestion
* model
* texture
* material
* animations (if export animations were selected)

```

File: BlockBench.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+To be able to spawn the model in the game you need to create an entire Minecraft legends entity. The following files are required for a Minecraft legends entity:
+**Behavior pack files**:
+* Entity.json
+**Resource Pack Files**:
+* Entity.json
+* Animation_controllers.json
+* Render_controllers.json
```
Comment (jsyarrow) [inline BlockBench.md:81]:
```suggestion
**Behavior pack files**:
* Entity.json

**Resource Pack Files**:
* Entity.json
* Animation_controllers.json
* Render_controllers.json

```

File: BlockBench.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+![](images/blockbench_doc/image08.png)
+![](images/blockbench_doc/image09.png)
+
+**Naming conventions**:
+Make sure to only use 
+* letters 
+* numbers
+* ```.``` (period)
+* ```-``` (dash)
+* ```_``` (underline)
```
Comment (jsyarrow) [inline BlockBench.md:99]:
```suggestion
* letters 
* numbers
* ```.``` (period)
* ```-``` (dash)
* ```_``` (underline) 

```

**Review by jsyarrow — COMMENTED** [review]
Trying again!

File: BlockBench.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+![](images/blockbench_doc/image07.png)
+
+When you have set your Export Options it will open a folder selection menu where you can select where the files will be exported. You have now successfully exported the following files:
+* model
+* texture
+* material
+* animations (if export animations were selected)
+
+To be able to spawn the model in the game you need to create an entire Minecraft legends entity. The following files are required for a Minecraft legends entity:
+**Behavior pack files**:
```
Comment (jsyarrow) [inline BlockBench.md:77]:
```suggestion

**Behavior pack files**:
```

**Pull request merged**